CONTRIBUTING.md

```diff
@@ -238,7 +238,7 @@ To build the documentation:
 
 1. Build and install PyTorch
 
-2. Install the prequesities
+2. Install the prerequisite
```

Comment:
nit: "Install the prerequisites"
(plural because there are multiple prerequisites)

Reply:
Am sorry I missed the plural thing!

Reply:
Oh no, don't apologize - thank you for finding the typo and fixing it!